src/Parallel/DataCollector.h

```diff
@@ -0,0 +1,86 @@
+#ifndef SEISSOL_PARALLEL_DATACOLLECTOR_H
+#define SEISSOL_PARALLEL_DATACOLLECTOR_H
+
+#include <cstddef>
+#include <vector>
+
+#include "Kernels/precision.hpp"
+
+#ifdef ACL_DEVICE
+#include "device.h"
+#endif
+
+namespace seissol::parallel {
+
+class DataCollector {
+  public:
+  DataCollector(const std::vector<real*>& indexDataHost,
```

Comment:
don't you mean indexedDataHost?

Reply:
It should mean "index data, acessible on the host."